.travis.yml

```diff
@@ -15,10 +15,9 @@ before_install:
 
 install:
   # Install composer packages, will also trigger dump-autoload
-  - composer self-update
-  - composer require phpunit/phpunit ^7
-  - composer require friendsofphp/php-cs-fixer
-  - composer install --no-interaction --ignore-platform-reqs
+  - composer validate
```

Comment:
Why did you move the `composer validate` command to the `install` step? It's part of the actual test suite, so I think it should run later as a `script`.

I agree on the other changes.

Reply:
I think and invalid composer config is a road block, so it should be number 1.

Reply:
That's right. Our Composer config is rarely completely invalid, but when it is it should fail immediately.